internal/pkg/aws/apprunner/apprunner.go

```diff
@@ -72,6 +73,18 @@ func (a *AppRunner) DescribeService(svcARN string) (*Service, error) {
 	}
 	sort.SliceStable(envVars, func(i int, j int) bool { return envVars[i].Name < envVars[j].Name })
 
+	var observabilityConfiguration ObservabilityConfiguration
+	if resp.Service.ObservabilityConfiguration != nil && aws.BoolValue(resp.Service.ObservabilityConfiguration.ObservabilityEnabled) {
+		if out, err := a.client.DescribeObservabilityConfiguration(&apprunner.DescribeObservabilityConfigurationInput{
+			ObservabilityConfigurationArn: resp.Service.ObservabilityConfiguration.ObservabilityConfigurationArn,
+		}); err == nil {
+			// NOTE: swallow the error otherwise, because observability is an optional description of the service.
+			// Example error: when "EnvManagerRole" doesn't have the "apprunner:ObservabilityConfiguration" permission.
```

Comment:
If they dont have `apprunner:ObservabilityConfiguration`, can this condition be true? 🤔

```golang
aws.BoolValue(resp.Service.ObservabilityConfiguration.ObservabilityEnabled)
```

Reply:
Typo!! I meant to say `...doesn't have the "apprunner:DescribeObservabilityConfiguration" permission.` i.e. the permission to make that API call.

Reply:
Hmm but jf we keep doing env upgrade their env manager role should always be able to describe

Reply:
It's possible though that they haven't run `copilot svc deploy` before running `copilot svc show`

Reply:
Oh that makes sense!